docs/src/prototypes/chatMessageWithPopover/Popover.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import { Menu, Popup, toolbarBehavior, popupFocusTrapBehavior } from '@stardust-ui/react'
+
+export interface PopoverProps {
+  className?: string
+}
+
+class Popover extends React.Component<PopoverProps> {
+  render() {
+    return (
+      <Menu
+        styles={{
+          transition: 'opacity 0.2s',
+          position: 'absolute',
+          top: '-20px',
+          right: '5px',
+          background: '#fff',
+          boxShadow: '0px 2px 4px #ddd',
+          borderRadius: '.3rem',
+          opacity: 0,
+
+          '& .smile-emoji': {
+            display: 'none',
+          },
+
+          '&.focused .smile-emoji': {
+            display: 'flex',
+          },
+
+          '&:hover .smile-emoji': {
+            display: 'flex',
+          },
+
+          '& a:focus': {
+            textDecoration: 'none',
+            color: 'inherit',
+          },
+
+          '& a': {
+            color: 'inherit',
+          },
+        }}
+        iconOnly
+        className={this.props.className}
+        items={[
+          { key: 'smile', icon: 'smile', className: 'smile-emoji' },
+          { key: 'smile2', icon: 'smile', className: 'smile-emoji' },
+          { key: 'smile3', icon: 'smile', className: 'smile-emoji' },
+          { key: 'a', icon: 'thumbs up' },
+          { key: 'c', icon: 'ellipsis horizontal' },
+        ]}
+        renderItem={renderContextMenu}
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+        accessibility={toolbarBehavior}
+        data-is-focusable={true}
+      />
+    )
+  }
+}
+
+export default Popover
+
+const renderContextMenu = (MenuItem, props) => {
+  if (props.icon !== 'ellipsis horizontal') {
+    return <MenuItem {...props} />
+  }
+
+  return (
+    <Popup
+      key={props.key}
+      position="below"
+      accessibility={popupFocusTrapBehavior}
+      trigger={<MenuItem {...props} />}
+      content={
+        <div
```

Comment:
@kuzhelov @mnajdova @layershifter @miroslavstastny 
should we reopen the discussion for exporting a blank component for styling (our `Slot` -> `Component`)?

@sophieH29 could use `createComponent` but it's a bit odd to tell users to create a blank component just so they can style things or use it as wrapper / container for other components

Reply:
1. We're going to drop `styles` prop at all, https://github.com/stardust-ui/react/pull/406#issuecomment-441041119
2. What is a benefit of usage `Slot` in this case?

Reply:
> We're going to drop styles prop at all, #406 (comment)
oh, right..

> What is a benefit of usage Slot in this case?
I just realized we're going to drop styles so I am not sure right now.

@layershifter @sophieH29 disregard my comment, pls and leave code as is; I'll close this conversation and open a new comment to express my query